### ISSUE TITLE
Zeit in DEP chronologisch

### ISSUE THREAD

**Issue by flostony** [a-sit-plus/at-registrierkassen-mustercode#636]
Ist es wichtig das die Belegzeit, gleich wie die Belegnummer chronologisch im DEP sein muss?
Beim Arbeiten mit mehreren Clients und einem DEP kann es vorkommen dass die Uhrzeit nicht exakt synchron ist und somit folgender Fall zustande kommt:

BELEG 1 mit TimeStamp 23.11.2017 08:00:15 generiert auf Client 1
BELEG 2 mit TimeStamp 23.11.2017 08:00:01 generiert auf Client 2 

Das hier zur Verfügung gestellt Prüftool bringt keinen Fehler/Warnung, jedoch wurde
uns von einem Prüfer mitgeteilt, dass seine Software hier einen Fehler oder Warnung? bringt.

**Comment by ztp-mino** [comment]
https://github.com/a-sit-plus/at-registrierkassen-mustercode/issues/144

**Comment by jirihelmich** [comment]
Are you sure that the a-sit-plus#144 still holds? Seen this issue as well with the new version of the validation tool, the bill date took priority over ordering of bills in the DEP export.

**Comment by ztp-mino** [comment]
No, I am not sure. I don't know what caused that change of heart either. The new version of the validation tool checks for the right order (by date) now, but I don't know if non-increasing dates should be considered an error.

**Comment by tomw1808** [comment]
So, we also went with #144 . Now some of our DEPs are invalid when the validation tool 1.1.1 is used. Essentially this would render our whole cashbox system useless and probably many others which are not just simple systems but maybe connected to ERP systems or other client/server architectures with several components.

Is there any final decision which was made? Is this even interesting for the BMF when they validate the DEP?

**Comment by AxelKutschera** [comment]
With the validation tool of the tax authority, a chronologically incorrect order of receipts (not "bills") leads to an error message. These errors have to be explained to the tax auditor.
Incidentally, we use the time of the signature creation for the field "Document date-time".
